docs/src/app/components/pages/components/Stepper/Page.js

```diff
@@ -4,30 +4,28 @@ import CodeExample from '../../../CodeExample';
 import PropTypeDescription from '../../../PropTypeDescription';
 import MarkdownElement from '../../../MarkdownElement';
 
-import stepperReadmeText from './README';
+import stepperReadmeText from './README.md';
+import advancedReadmeText from './Advanced.md';
+import HorizontalLinearStepper from './HorizontalLinearStepper';
+import HorizontalLinearStepperCode from '!raw!./HorizontalLinearStepper';
+import HorizontalNonLinearStepper from './HorizontalNonLinearStepper';
+import HorizontalNonLinearStepperCode from '!raw!./HorizontalNonLinearStepper';
 import VerticalLinearStepper from './VerticalLinearStepper';
-import VerticalNonLinearStepper from './VerticalNonLinearStepper';
-import VerticalLinearStepperWithOptionalStep from './VerticalLinearStepperWithOptionalStep';
 import VerticalLinearStepperCode from '!raw!./VerticalLinearStepper';
-import VerticalLinearStepperWithOptionalStepCode from '!raw!./VerticalLinearStepperWithOptionalStep';
+import VerticalNonLinearStepper from './VerticalNonLinearStepper';
 import VerticalNonLinearStepperCode from '!raw!./VerticalNonLinearStepper';
-import HorizontalLinearStepper from './HorizontalLinearStepper';
-import HorizontalLinearStepperCode from '!raw!./HorizontalLinearStepper';
+import GranularControlStepper from './GranularControlStepper';
+import GranularControlStepperCode from '!raw!./GranularControlStepper';
+import CustomIcon from './CustomIcon';
+import CustomIconCode from '!raw!./CustomIcon';
+import HorizontalTransition from './HorizontalTransition';
+import HorizontalTransitionCode from '!raw!./HorizontalTransition';
 
+import stepCode from '!raw!material-ui/lib/Stepper/Step';
 import stepperCode from '!raw!material-ui/lib/Stepper/Stepper';
-import verticalStepCode from '!raw!material-ui/lib/Stepper/VerticalStep';
-import horizontalStepCode from '!raw!material-ui/lib/Stepper/HorizontalStep';
-
-
-const descriptions = {
-  verticalLinearStepper: 'The vertical linear stepper requires steps be completed in a specific order.',
-  verticalLinearStepperWithOptionalStep: 'Set the `optional` property to `true` for optional steps.' +
-    'Pass a custom label view through `stepLabel` property to indicate an optional step.',
-  verticalNonLinearStepper: 'For the vertical non-linear stepper, steps can be completed in any order.',
-  horizontalLinearStepper: 'The horizontal linear stepper acts the same as the vertical linear stepper. ' +
-  'The horizontal stepper does not support optional or non-linear steps at this time.',
-};
-
+import stepButtonCode from '!raw!material-ui/lib/Stepper/StepButton';
```

Comment:
> Do you want me to change all of it and completely remove the /lib path in the webpack config?

I think that we should aim for this.
But that sounds out of the scope of this PR. Could you just remove the `lib` pattern from the Stepper part of the documentation?
Thanks